src/cpcmx/data.f90

```diff
@@ -239,19 +239,19 @@ function solvent_name(solvent)
             solvent_name='tbutylbenzene'
         case('tetrachloroethene','c2cl4')
             solvent_name='tetrachloroethene' 
-        case('tetrahydrofuran')
+        case('tetrahydrofuran','THF')
             solvent_name='tetrahydrofuran'
         case('tetrahydrothiophenedioxide','sulfolan') 
             solvent_name='tetrahydrothiophenedioxide' 
-        case('tetralin')
+        case('tetralin','1,2,3,4-tetrahydronaphthalene')
             solvent_name='tetralin'
         case('toluene')
             solvent_name='toluene'
         case('tributylphosphate')
             solvent_name='tributylphosphate'
         case('triethylamine')
             solvent_name='triethylamine'
-        case('trimethylbenzene')
+        case('trimethylbenzene','1,2,3-trimethylbenzene')
```

Comment:
At the moment, 1,2,3-trimethylbenzene is just a guess. I know its not 1,3,5-trimethylbenzene because that is already defined. But I hadn't looked around enough yet to confirm that this isn't 1,2,4-trimethylbenzene.